src/styles/stars.css

```
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  height: 16px;
+}
+
+.stars__star {
+  margin-left: 4px;
+  width: 16px;
+  border: 0;
+  background-image: url("../images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star:nth-child(n),
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5),
+.stars--5 .stars__star:nth-child(n+6) {
```

that's pretty much redundant here
```suggestion
{
```